RadixWallet/Core/SharedModels/P2P/ConnectorExtension/P2P+ConnectorExtension+Request.swift

```diff
@@ -141,4 +146,23 @@ extension P2P.ConnectorExtension.Request {
 			}
 		}
 	}
+
+	public struct LinkClientInteractionResponse: Sendable, Hashable, Encodable {
+		public enum Discriminator: String, Sendable, Hashable, Encodable {
+			case linkClient
+		}
+
+		public let discriminator: Discriminator
+		public let publicKey: Exactly32Bytes
+		public let signature: HexCodable
```

Comment:
Maybe we should move this model to Sargon? At least fix the properties like this:
`HexCodable` should have been deleted, I looks like it was accidentally reintroduced, please delete it, I mean the type itself).

`signature` will always be an `Ed25519Signature` (Sargon model) right? So change to that.

`publicKey` will always be an `Ed25519PublicKey` (Sargon model), so please change to that :)